2 Tests are failing depending the environment

## Describe the bug

2 tests are failing on certain environments.

```
FAILED TESTS:
  Form Integrations
    × ".serializedValue" returns all non disabled fields based on form structure
      HeadlessChrome 83.0.4103 (Windows 10.0.0)
    AssertionError: expected { Object (full_name, date, ...) } to deeply equal { Object (bio, checkers[], ...) }
      + expected - actual
         "checkers[]": [
           []
         ]
         "comments": ""
      -  "date": "2000-12-11"
      -  "datepicker": "2020-12-11"
      +  "date": "2000-12-12"
      +  "datepicker": "2020-12-12"
         "dinosaurs": ""
         "email": ""
         "favoriteColor": "hotpink"
         "full_name": {
    at Context.<anonymous> (packages/form-integrations/test/form-integrations.test.js:9:39)
  parseAmount()
    × parseAmount with locale set and length is more than four
      HeadlessChrome 83.0.4103 (Windows 10.0.0)
    AssertionError: expected 6 to equal 6000
      + expected - actual
      -6
      +6000
    at Context.<anonymous> (packages/input-amount/test/parsers.test.js:158:10)
    at Test.Runnable.run (node_modules/karma-mocha-snapshot/lib/adapter.js:14:17)
```

## To Reproduce

It's hard to know how to reproduce it, it seems that my local environment is making the tests failing. It could be something about timezone and locale settings.

## Additional context

My current locale is canadian french and my timezone is Eastern Daylight Time (GMT-4).

--- a/packages/form-integrations/test/helpers/umbrella-form.js
+++ b/packages/form-integrations/test/helpers/umbrella-form.js
@@ -44,13 +44,13 @@ export class UmbrellaForm extends LitElement {
           <lion-input-date
             name="date"
             label="Date of application"
-            .modelValue="${new Date('2000-12-12')}"
+            .modelValue="${new Date('2000/12/12')}"
             .validators="${[new Required()]}"
           ></lion-input-date>
           <lion-input-datepicker
             name="datepicker"
             label="Date to be picked"
-            .modelValue="${new Date('2020-12-12')}"
+            .modelValue="${new Date('2020/12/12')}"
             .validators="${[new Required()]}"
           ></lion-input-datepicker>
           <lion-textarea

--- a/packages/input-amount/test/parsers.test.js
+++ b/packages/input-amount/test/parsers.test.js
@@ -153,7 +153,7 @@ describe('parseAmount()', () => {
   it('parseAmount with locale set and length is more than four', () => {
     expect(
       parseAmount('6,000', {
-        locale: 'gb-GB',
+        locale: 'en-GB',
       }),
     ).to.equal(6000);
     expect(

